app/models/miq_worker/runner.rb

```diff
@@ -456,6 +484,17 @@ def set_process_title
 
   private
 
+  def worker_options
+    {
+      :messaging => MiqQueue.messaging_client_options,
+      :settings  => Settings.to_hash
```

Comment:
Are these the only settings we need for all of the provider workers?

Reply:
Not for provider workers, but I figured we might have non-provider non-rails workers in the future.  The worker_options from the example https://github.com/ManageIQ/manageiq/pull/22114#issuecomment-1251229243 shows where we pass in the ems, endpoint, and authentication info for providers.  We can add this to the base-manager worker classes so all the provider workers inherit it